core/state_processor.go

```diff
@@ -78,9 +78,13 @@ func (p *StateProcessor) Process(block *types.Block, statedb *state.StateDB, pri
 		misc.ApplyDAOHardFork(statedb)
 	}
 	blockContext := NewEVMBlockContext(header, p.bc, nil)
-	vmenv := vm.NewEVM(blockContext, vm.TxContext{}, statedb, p.config, cfg)
```

Comment:
Here is the main issue with the new way in geth : they use one evm instance, use a reset function to pass new tx context, but in goq, we have the privateStateDB that change on each tx
I choose to keep the old way, one evm per tx

Reply:
what happened here? as is outdated

Reply:
for reference to understand the main trouble in the new state_processor of geth